src/lib/components/content/BuildsList.svelte

```diff
@@ -20,8 +20,12 @@
 </header>
 
 <div class="block list">
-  <!-- eslint-disable-next-line svelte/require-each-key -->
-  {#each builds as build}
+  {#if !builds.length}
+    <center>
+      <p class="no-content"><i>There's nothing here... yet</i></p>
+    </center>
+  {/if}
+  {#each builds as build (build.id)}
     <Build {build} />
   {/each}
```

Comment:
Here's a fun Svelte trick, you can :else an #each list

```suggestion
  {#each builds as build (build.id)}
    <Build {build} />
  {:else}
    <center>
      <p class="no-content"><i>There's nothing here... yet</i></p>
    </center>
  {/each}
```